# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#result definition
shape_msgs/MeshTriangle[] triangles_CAD
geometry_msgs/Point[] vertices_CAD
geometry_msgs/Pose[] trajectory
float32 error
